app/views/events/participants.html.haml

```diff
@@ -0,0 +1,24 @@
+.col-md-12
+  %h3 Participants of #{@event.name}
+  %br
+  %table.table
+    %tr
+      %th #
+      %th Nickname
+      %th Email
+      %th State
+      %th Role
+      %th Change role to event organizer
+    - User.includes(:requests, :profile).where(requests: {event: @event}).each.with_index(1) do |user,index|
```

Comment:
While thinking about it: What about having a has_many through association in Request
http://guides.rubyonrails.org/association_basics.html#the-has-many-through-association

and then you can do in the controller 

```
@users = @event.users.distinct
```

Caution: Untested and please check if the distinct is necessary